stdlib/public/TensorFlow/Gradients.swift

```diff
@@ -544,6 +544,14 @@ extension Tensor where Scalar : TensorFlowFloatingPoint {
     })
   }
 
+  @inlinable
+  func _vjpSqueezingShape(at axes: [Int32]) -> (Tensor, (Tensor) -> Tensor) {
+    let value = squeezingShape(at: axes)
+    return (value, { v in
+      return v.reshaped(toShape: self.shapeTensor)
```

Comment:
* For better memory efficiency, it would be nice to not capture `self` in the pullback closure when it's not needed. 
* Omit `return` when a closure body has only one expression.

You can define the pullback this way
```swift
return (value, { [shape = shapeTensor] v in
  v.reshaped(toShape: shape)
})
```

Reply:
I'll make that change and make _vjpReshaped and _vjpExpandingShape consistent as well. There seem to be a couple different idioms used in this file.

Reply:
Thanks!